docs/en/wallet-attestation.rst

```diff
@@ -478,7 +478,69 @@ The body of the Wallet Attestation JWT MUST contain:
+
+
+A Wallet Instance can obtain a Wallet Attestation if it's in either `Installed`, `Operational` or `Valid` state; that implies that a `Deactivated` Wallet Instance cannot obtain a Wallet Attestation hence it cannot interact with other entities such as PID/(Q)EAA Providers and Relying Parties.
+
+States
+~~~~~~~~~~~~~~~~~~
++---------------+--------------------------------------------------------------------------------------------------------------------------------+
+| State         | Description                                                                                                                    |
++===============+================================================================================================================================+
+| `Installed`   | The Holder has installed the Wallet Solution on the device                                                                     |
++---------------+--------------------------------------------------------------------------------------------------------------------------------+
+| `Operational` | The Wallet Instance has been verified and the Wallet Hardware Key has been registered; no valid PID is present in the storage. |
++---------------+--------------------------------------------------------------------------------------------------------------------------------+
+| `Valid`       | A valid PID is present in the storage                                                                                          |
++---------------+--------------------------------------------------------------------------------------------------------------------------------+
+| `Deactivated` | The Wallet Instance has been revoked and its Wallet Hardware Key has been marked as not usable                                 |
++---------------+--------------------------------------------------------------------------------------------------------------------------------+
+
+Transitions
+~~~~~~~~~~~~~~~~~~
+.. list-table::
+   :widths: 20 60
+   :header-rows: 1
+
+   * - **Transition**
+     - **Description**
+   * - `install`
+     - The User performs a fresh installation or restores the initial state of the Wallet Instance on the device.
+   * - `verify`
+     - The Wallet Instance has been verified by the Wallet Provider and its Wallet Hardware Key has been registered.
+   * - `validate`
+     - The Wallet Instance obtains a valid PID.
+   * - `invalidate`
+     - The PID expires or gets revoked.
+   * - `revoke`
+     - The Wallet Provider marks the Wallet Instance as not usable.
+   * - `uninstall`
+     - The User removes the Wallet Instance from the device.
+
+Revocations
+~~~~~~~~~~~~~~~~~~
+As mentioned in the *Wallet Instance initialization and registration* section above, a Wallet Instance is bound to a Wallet Hardware Key and it's uniquely identified by it. 
```

Comment:
It is better to clarify that only the Wallet Provider is able to identify the Wallet Instance by the Wallet Hardware Key uniquely. It is important to avoid confusion on this aspect.

Reply:
Added in [4796bc4](https://github.com/italia/eudi-wallet-it-docs/pull/242/commits/4796bc4078eeb2a85d5f8e0f77307ae347671e2a)

> Please note that the Wallet Instance shares the its Wallet Hardware Key only with the Wallet Provider, thus the Wallet Provider is the only entity identify a Wallet Instance by it.